### DISCUSSION TITLE
What is the current health of beets?

### DISCUSSION THREAD

**Discussion by Ashe** [beetbox/beets#4562] [General]
I used to love and use beets without issue, however today has been riddled with issues after returning to beets after ~2 years.

Here's a summary of my issues:

* Files are being duplicated
![image](https://user-images.githubusercontent.com/8940077/204109746-4b193bf7-fd2e-4138-8971-19ed2a42df4c.png)

* Entire albums are being duplicated
![image](https://user-images.githubusercontent.com/8940077/204109767-413f557a-d744-418f-aacc-b2e25c199d23.png)

* `beet update` is ruining metadata
![image](https://user-images.githubusercontent.com/8940077/204109795-53e3652b-1d4f-4550-94b7-aee7ffeaf100.png)

* `beet move` doesn't do anything despite saying it is / feels like it always needs moving?
![image](https://user-images.githubusercontent.com/8940077/204109811-67469259-2dd4-47b7-8eb3-4c6edbb149fd.png)

* All of my tracks were placed into a single album with `beet move`, metadata not affected
* Then, `beet update` and `beet write` screwed up the metadata (this was user error, I thought it was just the folders that was broken, so refreshing the metadata would correct album names)

I've spent hours blaming myself and spiralling because I used to use this program with little to no care and my library was perfectly organised and tagged.. now though its a mess and because I didn't think this would happen I didn't think to back it up. My mistake but I'm just hoping its recoverable.

Is the general health of `beets` not great right now? I don't know how things can go catastrophically wrong :( having a bad night, sorry.

**Comment by arsaboo** [comment]
Some perspective here - https://github.com/beetbox/beets/discussions/4558

The `beet update` bug has been filed - https://github.com/beetbox/beets/issues/4528

**Comment by Ashe** [comment]
Yeah, I apologise, I think it's just been a long night. I'm still appreciative of all the good things beets has done for me in the past, in fact it's because I think it's so great that it hurts more when I get issues with it!

**Comment by arsaboo** [comment]
I know and we all feel the pain. I am sure the community will step forward to keep the project alive.

**Comment by sampsyo** [comment]
I don't have too much to add, except that I'm sorry you ran into this cluster of (all very frustrating!) issues. Thanks for your patience with the project, and I hope you'll stick with it despite the challenges.
